python/tests/res/enkf/data/test_enkf_config_node.py

```diff
@@ -0,0 +1,17 @@
+import os.path
+import json
+
+from res.enkf.config import EnkfConfigNode
+from ecl.test import TestAreaContext, ExtendedTestCase
+
+
+class EnkfConfigNodeTest(ExtendedTestCase):
+
+    def test_gen_data(self):
+
+        # Must have %d in filename argument
+        with self.assertRaises(ValueError):
+            config_node = EnkfConfigNode.create_gen_data( "KEY", "FILE" )
+
+        config_node = EnkfConfigNode.create_gen_data( "KEY", "FILE%d" )
+        self.assertIsInstance( config_node, EnkfConfigNode )
```

Comment:
Could you please also verify the values in this test, both for the default `report_step` and for `default_step=range(10)` or something similar?